src/connectors/simulatorradio.ts

```diff
@@ -0,0 +1,9 @@
+export {};
+
+Connector.playerSelector = '.container.player-wrap';
+Connector.trackArtSelector = '.record-img';
+Connector.artistSelector = '.song>.Artist';
+Connector.trackSelector = '.song>.Title';
+Connector.playButtonSelector = '.playpause > .fa-play';
+
+Connector.onReady = Connector.onStateChanged;
```

Comment:
Don't think we need to include this as it's done automatically when the DOM loads